padertorch/ops/mappings.py

```diff
@@ -5,11 +5,34 @@
 
 __all__ = [
     'ACTIVATION_FN_MAP',
-    'POOLING_FN_MAP',
-    'DTYPE_MAP',
 ]
 
-ACTIVATION_FN_MAP = Dispatcher(
+class CallableDispatcher(Dispatcher):
+    """
+       Is basically a dict for non-callable inputs
```

Comment:
This formulation is confusing. First time I read it I thought "Why is the CallableDispatcher not used for callable inputs?". I think the other way round, like "For non-callable inputs, ... . For callable inputs, ...", is less confusing

Reply:
done